app/controllers/projects_controller.rb

```diff
@@ -214,4 +215,7 @@ def set_project
     authorize @project
   end
 
+  def prepare_session
+    session[:import_job] = session[:import_job].with_indifferent_access unless session[:import_job].nil?
```

Comment:
an alternative could be
```ruby
def prepare_session
  session[:import_job] = (session[:import_job] || {}).with_indifferent_access
end
```

but [here](https://github.com/Codeminer42/cm42-central/pull/101/files#diff-757cba42ec9599bac53d4cc33cdde3bfR95) you'll need to use `if session[:import_job].present?`

it's not a blocker, just 💅